client/src/app/components/Layout/Layout.jsx

```diff
@@ -162,8 +162,8 @@ export class Layout extends PureComponent {
               title={formatMessage({
                 id:
                   viewType === ViewType.LIST
-                    ? 'app.layout.list-view'
-                    : 'app.layout.tile-view'
+                    ? 'app.layout.tile-view'
```

Comment:
??

Reply:
There was a bug. Message-ids were reversed. Now it's ok. If there is a list view we want to change it to a tile view. So button title should be: 'change to tile view'

Reply:
Consider renaming locale key then, i.e. "app.layout.switch-to-tile-view".